src/blk/pmem/PMEMDevice.cc

```diff
@@ -54,10 +59,59 @@ int PMEMDevice::_lock()
   return 0;
 }
 
+static int pmem_check_file_type(int fd, const char *pmem_file, uint64_t *total_size)
+{
+  int rc = 0;
+  struct stat file_stat;
+  FILE *sfile;
```

Comment:
please define a variable closer to where it is used for better readability.

Reply:
ack